packages/@react-types/numberfield/src/index.d.ts

```diff
@@ -31,7 +32,8 @@ export interface NumberFieldProps extends InputBase, Validation, FocusableProps,
   formatOptions?: Intl.NumberFormatOptions
 }
 
-export interface AriaNumberFieldProps extends NumberFieldProps, DOMProps, AriaLabelingProps {
+export interface AriaNumberFieldProps extends NumberFieldProps, DOMProps, AriaLabelingProps,
+  Omit<TextInputDOMProps, 'autoComplete' | 'inputMode' | 'maxLength' | 'minLength' | 'pattern' | 'placeholder' | 'type'> {
```

Comment:
does this follow any patterns they already have?

Reply:
My original inclination was to just do a `Pick<TextInputDOMProps, 'name'>`, but I didn't find any other usages of `Pick` in the `@react/types` definitions; `Omit` was usually used in these cases so that's why I went this route (but admittedly it does feel clunky to omit so many properties).  

I wanted to still pull from `TextInputDOMProps` though (rather than redefining the individual properties in `AriaNumberFieldProps`), since the props in `useNumberField` pass to `useFormattedTextField` (and `TextInputDOMProps` is part of the definition for the input props to that function).  Mostly I liked that it could pass the additional events from `TextInputDOMProps` (like `onSelect` or `onInput`), but I'm not sure if it's 100% worth it to go this route.  I'd definitely love guidance from the reviewers on what the preferred path is in this type of case!